AppCenter/AppCenterTests/MSAuthTokenContextTests.m

```diff
@@ -34,15 +34,43 @@ - (void)testSetAuthToken {
 
   // If
   NSString *expectedAuthToken = @"authToken1";
+  NSString *expectedAccountId = @"account1";
+  NSString *expectedAccountId1 = @"account2";
```

Comment:
I know it's only a unit test, please consider make the variable name and it's value consistent, not self-contradicting. 😄